spec/api/v1/api_works_spec.rb

```diff
@@ -477,15 +488,9 @@
 end
 
 describe "API WorksController - Find Works" do
-  before do
-    @work = FactoryGirl.create(:work, posted: true, imported_from_url: "foo")
-  end
-
-  after do
-    @work.destroy if @work
-  end
-
   describe "valid work URL request" do
+    work = FactoryGirl.create(:work, posted: true, imported_from_url: "foo")
```

Comment:
This should be okay sans the `FactoryGirl.`

Reply:
Oh! Just thought of this: we have a [posted_work factory](https://github.com/otwcode/otwarchive/blob/bd4f323003a7f63d0e708ff334fd666e4760601a/factories/works.rb#L26) you could use here and on line 13 of spec/api/v2/api_works_search_spec.rb